sdk/typescript/src/types/common.ts

```diff
@@ -76,7 +76,7 @@ export function isValidTransactionDigest(
 // which uses the Move account address length
 // https://github.com/move-language/move/blob/67ec40dc50c66c34fd73512fcc412f3b68d67235/language/move-core/types/src/account_address.rs#L23 .
 
-export const SUI_ADDRESS_LENGTH = 20;
+export const SUI_ADDRESS_LENGTH = 32;
```

Comment:
cc @Jordan-Mysten i tried to make all the typescript changes too (at least all unit tests are passing), feel free to push any more